skycultures/indian/constellation_names.eng.fab

```diff
@@ -71,7 +71,9 @@ Abh	"Abhijit"	_("Victorious")
 Dhani	"Dhaniṣṭhā"	_("Wealthiest")
 # vAchaspatyam: शतं भिषज इव ताराः यत्र ।
 Śata	"Śatabhiṣak"	_("Hundred cures")	
-# Ancient name: प्रोष्ठपद
+
+# Ancient name: प्रोष्ठपद. प्रोष्ठ referred to a bench in some vedic literature, and later a bull/ ox.
```

Comment:
It would help non-Indians greatly if there was a transliteration to Latin characters following each Indian word.

Reply:
Done.